<launch>
    <arg name="output"            default="screen"    doc="display output to screen or log file"/>
    <arg name="TrajTopic"   default="/planning/trajectory"    doc="ROS topic for the reference trajectory"/>
    <arg name="PoseTopic"   default="/zed2/zed_node/odom"    doc="ROS topic for the pose"/>
    <arg name="ControllerTopic"   default="/rc_control"    doc="ROS topic to publish"/>
    
    <node pkg="traj_tracking_ros" type="traj_tracking_node.py" name="traj_tracking_node" output="$(arg output)" args="" required="true">
        <param name="TrajTopic"           value="$(arg TrajTopic)" />
        <param name="PoseTopic"           value="$(arg PoseTopic)" />
        <param name="ControllerTopic"           value="$(arg ControllerTopic)" />
        <param name="TrackParamsFile"     value="$(find traj_tracking_ros)/params/stanley.yaml" />
    </node>
    <node name="tracking_visualize_node" pkg="traj_tracking_ros" type="tracking_visualizer.py" />
</launch>
